lib/db/query.js

```diff
@@ -189,8 +189,62 @@ async function getTestBody(testId) {
   return json;
 }
 
+async function getTestsRuns(id) {
+  const testsQuery = `
+    SELECT 
+    t.name,
+    t.url,
+    t.created_at,
+    t.updated_at,
+    hm.display_name
+    FROM tests t
+    JOIN http_methods hm ON hm.id = t.method_id
+    WHERE t.id = ${id};`;
+
+  const runsQuery = `
+    SELECT DISTINCT
+      tr.id test_run_id,
+      r.id region_id,
+      r.display_name,
+      r.flag_url,
+      tr.pass,
```

Comment:
all rows in `test_runs` table have  `null` for `pass` currently. My latest [PR](https://github.com/No-Name-temporary/test-result-writer/pull/3) addresses this

Reply:
ok. That makes sense now! Thanks!